inspirehep/modules/workflows/receivers.py

```diff
@@ -66,4 +66,12 @@ def spawn_arXiv_workflow_from_oai_harvest(request, records, name, **kwargs):
         marcxml = convert(recxml, "oaiarXiv2marcxml.xsl")
         record = create_record(marcxml)
         hep_record = hep.do(record)
-        start.delay(workflow, data=[hep_record])
+        engine = WorkflowEngine.with_name(workflow)
+        engine.save()
+        objects = get_workflow_object_instances(data, engine)
```

Comment:
This function is not defined anywhere... and also `data`. And `objects` is not used... OK, it's WIP :) Was hoping to use it on production :dancer: